samples/arch/smp/pktqueue/sample.yaml

```diff
@@ -18,13 +18,5 @@ tests:
     depends_on:
       - smp
     filter: (CONFIG_MP_MAX_NUM_CPUS > 1)
-    platform_exclude:
```

Comment:
In the past this was needed. Now, all this boards are filtered out. No need to rename it to new board model. Instead, delete this.